MobileFoodSchedules/MobileFoodSchedules/Models/MobileFoodPermit.cs

```diff
@@ -1,12 +1,4 @@
-﻿// <auto-generated />
-//
-// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
-//
-//    using QuickType;
-//
-//    var mobileFoodPermit = MobileFoodPermit.FromJson(jsonString);
-
```

Comment:
Good to remove commented out code.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.